src/pages/contributors/valentin-delaye.adoc

```diff
@@ -0,0 +1,54 @@
+= Valentin Delaye
+:page-name: Valentin Delaye
+:page-linkedin: 
+:page-twitter: 
+:page-github: jonesbusy
+:page-email: 
+:page-image: avatar/valentin-delaye.png
+:page-pronouns: He/Him/His
+:page-location: Lausanne, Switzerland
+:page-firstcommit: 2009
```

Comment:
User since 2009 but first commit in 2018

Reply:
Out of scope for this PR but having both values could be nice.